modules/kubecf/install.sh

```diff
@@ -90,9 +108,20 @@ kubectl create secret generic -n scf susecf-scf.var-cf-admin-password --from-lit
 ## CF-Operator >= 4 don't have deployment name in front of secrets name anymore
 kubectl create secret generic -n scf var-cf-admin-password --from-literal=password="${CLUSTER_PASSWORD}"
 
+kubecf_install_args=(--values scf-config-values.yaml)
```

Comment:
This concerns me to be honest. So far, we had made a point on saving all the state that we are going to install in the values.yaml, and this departs from it. I think this is painful, and will bring us headaches in the future, both developing and debugging already deployed clusters, because if we get into the habit of setting deployment options inline, you never know on what state the cluster was deployed.

I think these changes should go in `modules/kubecf/gen-config.sh`. Yes, we lose functionality of airgapping "on the fly", but production deployments don't need to be airgapped "on the fly", you configure the deployment to be airgapped, and then you deploy.